mmcv/runner/hooks/lr_updater.py

```diff
@@ -165,7 +165,7 @@ def get_lr(self, runner, base_lr):
 @HOOKS.register_module()
 class StepLrUpdaterHook(LrUpdaterHook):
 
-    def __init__(self, step, gamma=0.1, **kwargs):
+    def __init__(self, step, gamma=0.1, min_lr=None, **kwargs):
```

Comment:
Unit tests should be added to verify this feature.

Reply:
Done.